### ISSUE TITLE
TCP/IP

### ISSUE THREAD

**Issue by polytronics** [Pacmanfan/UVDLPSlicerController#22]
Hi, do you think you can add TCP/IP connection to the printer in addition to the classic "com" port ?for this we need to provide an IP address and a port.

If you prefere, i can take a look and try to do it but i guess you can do it way faster and better than me.

**Comment by Pacmanfan** [comment]
Are you trying to treat the TCP/IP port basically as an external serial port?

**Comment by skapin** [comment]
yes. Same "protocol", just not trought USB/serial cable but LAN/internet cable. (or even localhost)
You need to instance a simple tcpConnector instead of serial and here you go.

This function is on repetierHost for the record
